backend/app/adapter/db/url.go

```diff
@@ -170,6 +171,101 @@ func (u URLSql) composeParamList(numParams int) string {
 	return parameterStr
 }
 
+// FindURLsByUser finds all urls created by a given user
+func (u URLSql) FindURLsByUser(user entity.User) ([]entity.URL, error) {
```

Comment:
For your information, we have `FindAliasesByUser(user entity.User) ([]string, error)` here: https://github.com/short-d/short/blob/cc1f83f1e1a39be792a5c62888991f75ad0b3ba7/backend/app/adapter/db/userurl.go#L38

Reply:
@byliuyang, we need to have longLink also to display table here: https://github.com/short-d/short/pull/615